internal/game/player.go

```diff
@@ -45,6 +45,7 @@ func (g *Game) moveForward() {
 	if g.gopher.p.Is(g.player.FrontPoint()) {
 		g.state = sWin
 	}
+	g.stats.Moves++
```

Comment:
Should we increment count on rotations too @necrophonic ?

Reply:
Hmm it's a good point I don't think so.

I think if you're counting "moves" I wouldn't count rotations as wouldn't want to penalise the player for having to turn at corners